.github/workflows/ci.yml

```diff
@@ -121,11 +118,26 @@ jobs:
       - name: Build third-party
         run: ./ci/before_script.sh
 
-      - name: Build and test
-        run: ./ci/script.sh
+      - name: Build
+        run: ./ci/run_tests.sh build
+
+      - if: matrix.flavor != 'tsan' && ( success() || failure() )
```

Comment:
Would using `!cancelled()` instead of `success() || failure()` make more sense?

Reply:
Yes. Thanks